File: tecnologias/java/maven/kata-01-legacy-code/src/main/java/org/bacraftmanship/model/Comercio.java
package org.bacraftmanship.model;

public class Comercio {

    private String producto;
    private double precioCompra;
    private int cantidad;

    public String compra(String producto, double precioCompra, int cantidad) {

        this.producto = producto;
        this.precioCompra = precioCompra;
        this.cantidad = cantidad;

        return producto;
    }

    public Double vender(String producto, double precioVenta) {

        if ( this.producto.equals(producto)) {
            return precioVenta - (precioCompra / cantidad);
        }

        return null;
    }
}

File: tecnologias/python/bacraftmanship/model/comercio.py
class Comercio:
    def compra(self, producto, precio_compra, cantidad):
        self.producto = producto
        self.precio_compra = precio_compra
        self.cantidad = cantidad
        return producto

    def vender(self, producto, precio_venta):
        if self.producto == producto:
            return precio_venta - (self.precio_compra / self.cantidad)
        return None
